deploy/charts/cert-manager/values.yaml

```diff
@@ -703,7 +703,11 @@ startupapicheck:
 
   # Additional command line flags to pass to startupapicheck binary.
   # To see all available flags run docker run quay.io/jetstack/cert-manager-ctl:<version> --help
-  extraArgs: []
+  #
+  # We enable verbose logging by default so that if startupapi check fails users
+  # can know what exactly caused the failure.
+  extraArgs:
+  - --verbose
```

Comment:
I don't think the `--verbose` flag exists.
```
error: unknown flag: --verbose
```

You can use `--v=...` instead.

eg.:
```
$ _bin/cmctl/cmctl-linux-amd64 check api --v=6
I1117 11:08:59.519276  532385 loader.go:395] Config loaded from file:  /home/tramlot/.kube/config
I1117 11:08:59.534300  532385 round_trippers.go:553] GET https://127.0.0.1:43857/api 200 OK in 14 milliseconds
I1117 11:08:59.535585  532385 round_trippers.go:553] GET https://127.0.0.1:43857/apis 200 OK in 0 milliseconds
I1117 11:08:59.537509  532385 round_trippers.go:553] GET https://127.0.0.1:43857/apis/cert-manager.io/v1 200 OK in 1 milliseconds
I1117 11:08:59.541878  532385 round_trippers.go:553] POST https://127.0.0.1:43857/apis/cert-manager.io/v1/namespaces/default/certificates?dryRun=All 201 Created in 3 milliseconds
The cert-manager API is ready
```

Reply:
Doh! I must have an old version on my laptop:

```
$ ctl check api --help
This check attempts to perform a dry-run create of a cert-manager v1alpha2 Certificate resource in order to verify that CRDs are installed and all the required webhooks are reachable by the K8S API server. We use v1alpha2 API to ensure that the API server has also connected to the cert-manager conversion webhook.

Usage: cmctl

Flags:
...
  -v, --verbose                        Print detailed error messages
      --wait duration                  Wait until the cert-manager API is ready (default 0s)
```

Reply:
Done.